pkg/proc/breakpoints.go

```diff
@@ -463,15 +463,51 @@ func (t *Target) SetBreakpoint(addr uint64, kind BreakpointKind, cond ast.Expr)
 	return t.setBreakpointInternal(addr, kind, 0, cond)
 }
 
+// SetEBPFTracepoint will attach a uprobe to the function
+// specified by 'fnName'.
 func (t *Target) SetEBPFTracepoint(fnName string) error {
+	// Not every OS/arch that we support has support for eBPF,
+	// so check early and return an error if this is called on an
+	// unsupported system.
 	if !t.proc.SupportsBPF() {
 		return errors.New("eBPF is not supported")
 	}
+	// Start putting together the argument map. This will tell the eBPF program
+	// all of the arguments we want to trace and how to find them.
 	var args []ebpf.UProbeArgMap
 	fn, ok := t.BinInfo().LookupFunc[fnName]
 	if !ok {
 		return fmt.Errorf("could not find function %s", fnName)
 	}
+
```

Comment:
I think `findType` and then iterating through the `Field` slice is better.

Reply:
Done.